tutorial/README.md

```diff
@@ -458,6 +459,33 @@ INSERT INTO conditions VALUES (now(), 'Prague', 30, 50);
 docker-compose -f docker-compose-timescaledb.yaml down
 ```
 
+## Using Informix
+
+```shell
+# Start the topology as defined in https://debezium.io/documentation/reference/stable/tutorial.html
+export DEBEZIUM_VERSION=2.5
+
+docker-compose -f docker-compose-ifx.yaml up --build
+
+# Start Informix connector
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @register-ifx.json
+
+# Consume messages from a Debezium topic
+docker-compose -f docker-compose-ifx.yaml exec kafka /kafka/bin/kafka-console-consumer.sh \
+    --bootstrap-server kafka:9092 \
+    --from-beginning \
+    --property print.key=true \
+    --topic ifxserver.informix.customers
+
+# Modify records in the database via DB2 client
+docker-compose -f docker-compose-ifx.yaml exec ifxserver bash -c 'dbaccess informix -'
```

Comment:
Unfortunately this command does not work. You need to use sysuser instead of informix but still at the end it fails with
```
-23101 Unable to load locale categories.
```

Reply:
Yes, but the SQL passes for me(?)

Reply:
OK,
I don't get the locale problem, even when I use the literal commands.
Database name is obvious. 
There was a problem with the exec path to dbaccess depending on whether I run from windows or linux (again) but with a login shell (-l swithch added) it works for both.
These are fixed in my last commit...